src/main/java/redis/clients/jedis/Jedis.java

```diff
@@ -108,8 +108,8 @@ public String get(final String key) {
   }
 
   /**
-   * Test if the specified key exists. The command returns the number of keys existed
-   * Time complexity: O(N)
+   * Test if the specified key exists. The command returns the number of keys existed Time
```

Comment:
I don't think it becomes a reason to fix the formatter. But if we can fix the formatter to not modify comments, it may be also fine to me.